_tools/check_docs.ts

```diff
@@ -497,8 +403,8 @@ function assertConstructorDocs(
  * Checks a module document for:
  * - Code snippets that execute successfully.
  */
-function assertModuleDoc(document: DocNodeWithJsDoc<DocNodeModuleDoc>) {
-  assertSnippetsWork(document.jsDoc.doc!, document);
+function assertModuleDoc(_document: DocNodeWithJsDoc<DocNodeModuleDoc>) {
+  // assertSnippetsWork(document.jsDoc.doc!, document);
```

Comment:
Can this be enabled?

Reply:
I disabled that for now, because there are many mods that do not provide an example where the `@module` is declared. These are:
```sh
[error] @example tag must have a TypeScript code snippet at file:///Users/timreichen/Sites/std/crypto/crypto.ts:4
[error] @example tag must have a TypeScript code snippet at file:///Users/timreichen/Sites/std/encoding/base64url.ts:4
[error] @example tag must have a TypeScript code snippet at file:///Users/timreichen/Sites/std/cli/parse_args.ts:4
[error] @example tag must have a TypeScript code snippet at file:///Users/timreichen/Sites/std/http/etag.ts:4
[error] @example tag must have a TypeScript code snippet at file:///Users/timreichen/Sites/std/http/file_server.ts:7
[error] @example tag must have a TypeScript code snippet at file:///Users/timreichen/Sites/std/http/negotiation.ts:4
[error] @example tag must have a TypeScript code snippet at file:///Users/timreichen/Sites/std/http/status.ts:4
[error] @example tag must have a TypeScript code snippet at file:///Users/timreichen/Sites/std/http/user_agent.ts:8
8 errors found
```
I am not sure how to handle that. Does the module need examples in addition to examples for exports?

Reply:
I believe this check is for showing the example in the top page of the package in JSR website (e.g. https://jsr.io/@std/encoding ), and those errored files are not for the top level of the package. I suggest we should do something like:
```suggestion
  if (document.location.filename.endsWith("/mod.ts")) {
    assertSnippetsWork(document.jsDoc.doc!, document);
  }
```

Reply:
Done